### ISSUE TITLE
webmin command not included in package

### ISSUE THREAD

**Issue by swelljoe** [webmin/webmin#1248]
I think the `bin` directory and `webmin` script would be useful additions to the package (linking `webmin` to `/sbin/webmin` would be ideal, but even just including the `bin` dir in the bundle would be great). Is there anything stopping that?

**Comment by iliarostovtsev** [comment]
I think it's a great idea, Joe, I was also always wondering why is it missing!

https://github.com/webmin/webmin/commit/f72f7eca8d0d710b4de49a11cdc1c4f3e00a87cd

I don't think, if it's necessary to add those to the `path`?

**Comment by swelljoe** [comment]
No, the only thing that needs to be in the path is the `webmin` command. It can read the Webmin configuration to figure out where its friends live. At least, that's how it's meant to work, and I think I tested it, but it's been ages since I looked at it. I forgot all about it after merging it, until I wanted to tell someone to use it, and it wasn't there! (2FA and ports and such come up all the time, and we don't have an easy "no editing obscure config files" method to deal with them.)

As an aside, we should test to make sure your language scripts are compatible with being subcommands of `webmin`, since they'll show up that way if the whole dir gets bundled). If they don't have usage/POD docs in a form that works that'd be something to fix. Though I wonder if maybe we ought to exclude them from the packages since they're developer-focused tools.

**Comment by iliarostovtsev** [comment]
Okay, thanks, I've checked and fixed few issues.

https://github.com/webmin/webmin/commit/918645c7a5a3dd7e897a49ba492630370148eecf
https://github.com/webmin/webmin/commit/fa717574b07fc47a533154c53ab2e4df7387fff8
https://github.com/webmin/webmin/commit/870893521e7a5d199d344fcda8eaa2715887e2ab
https://github.com/webmin/webmin/commit/ec29744a4ee0e89e18114ab1bc9369e7fb976eed

> Though I wonder if maybe we ought to exclude them from the packages since they're developer-focused tools.

I don't know. It's a hard question as Webmin itself is not a toy for kids.

**Comment by iliarostovtsev** [comment]
Additionally, once we're doing things for developers, I added `update-devel` script for making things a bit more easy.

It's explicitly saying that it's development and unstable, and you would need to know things, in order to use a command anyway.

~~~POD
Name:
    update-devel

Description:
    Update Webmin/Usermin and/or Authentic Theme from Git, with latest
    unstable, development version.

Usage:
    update-devel [options] --product <webmin|usermin> [--theme]

Options:
    --product, -p
        Specify product name to update, like "webmin" or "usermin".

    --theme, -t
        If set, only Authentic Theme will be updated.

    --help, -h

~~~

**Example of usage** -


 - update Webmin from Git: 

~~~
webmin update-devel --product webmin
~~~

 - only update Authentic Theme for Usermin.

~~~
webmin update-devel --product usermin --theme
~~~


https://github.com/webmin/webmin/commit/669b00589fc108283c4d6f19baae73bb400bc09b

@jcameron Do you think it will be possible to symlink `/usr/{libexec|share}/webmin/bin/webmin` to `/usr/sbin/webmin` upon package installation?
